generate/struct.go

```diff
@@ -418,8 +418,12 @@ func (ef *extractedField) goType(required, pointers bool) string {
 			customType = true
 		}
 	case "object":
-		// This only happens with nested structures.
-		goType = "struct"
+		if len(ef.fields) == 0 {
+			goType = "map[string]string"
```

Comment:
I'd add a TODO comment about using the `additionalProperties` JSON schema value to more intelligently create the target type in the future.